server/simple_server.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/gorilla/context"
 	"github.com/gorilla/mux"
 	"github.com/rcrowley/go-metrics"
+
```

Comment:
Mind removing this blank line? These imports can be grouped with the other third party packages.